docs/source/introduction/communication.rst

```diff
@@ -116,6 +116,15 @@ start communicating as follows::
 Here we use `'\n'` known as 'line feed'. This is a common value, another one is
 `'\r'` i.e. 'carriage return', and in some cases the null byte '\0' is used.
 
+For instruments that communicate over serial, you need to ensure you configure
```

Comment:
I would use `.. note::` for this section since it is serial specific. I would also mention teh existence of other parameters and refer people to the API docs for complete details.